superset/daos/css.py

```diff
@@ -15,31 +15,12 @@
 # specific language governing permissions and limitations
 # under the License.
 import logging
-from typing import Optional
-
-from sqlalchemy.exc import SQLAlchemyError
 
 from superset.daos.base import BaseDAO
-from superset.daos.exceptions import DAODeleteFailedError
-from superset.extensions import db
 from superset.models.core import CssTemplate
 
 logger = logging.getLogger(__name__)
 
 
 class CssTemplateDAO(BaseDAO):
     model_cls = CssTemplate
-
-    @staticmethod
```

Comment:
Copypasta except the `BaseDAO` is defined as a `classmethod` as opposed to a `staticmethod`.